src/app/AppLayout/AppLayout.tsx

```diff
@@ -220,27 +233,38 @@ const AppLayout: React.FC<AppLayoutProps> = ({ children }) => {
   const dismissSslErrorModal = React.useCallback(() => setShowSslErrorModal(false), [setShowSslErrorModal]);
 
   const onNavToggleMobile = React.useCallback(() => {
-    setIsNavOpenMobile((isNavOpenMobile) => !isNavOpenMobile);
-  }, [setIsNavOpenMobile]);
+    setIsNavOpen((isNavOpenMobile) => !isNavOpenMobile);
+  }, [setIsNavOpen]);
 
   const onNavToggle = React.useCallback(() => {
-    setIsNavOpen((isNavOpen) => !isNavOpen);
-  }, [setIsNavOpen]);
+    setIsNavOpen((isNavOpen) => {
+      if (joyState.run === true && joyState.stepIndex === 1 && !isNavOpen) {
+        setJoyState({ stepIndex: 2 });
+      }
+      return !isNavOpen;
+    });
+  }, [setIsNavOpen, joyState, setJoyState]);
 
+  // prevent page resize to close nav during tour
   const onPageResize = React.useCallback(
     (props: { mobileView: boolean; windowSize: number }) => {
-      setIsMobileView(props.mobileView);
+      if (joyState.run === false) {
+        setIsMobileView(props.mobileView);
+        if (props.mobileView) {
+          setIsNavOpen(false);
```

Comment:
Seems like if u resize to smaller width, then resize back. The menu remains closed. Is this the desired behaviour now?

Reply:
Oh, my bad, I forgot what the old implementation was, I will revert that.